corehq/motech/repeaters/tasks.py

```diff
@@ -55,14 +56,18 @@
     run_every=crontab(day_of_month=27, hour=6, minute=0),
     queue=settings.CELERY_PERIODIC_QUEUE,
 )
-def clean_logs():
+def raw_delete_logs():
     """
     Drop MOTECH logs older than 90 days.
 
+    Does not notify pre_delete and post_delete signal listeners. Does
+    not respect cascade delete.
+
     Runs on the 27th of every month.
     """
-    ninety_days_ago = datetime.now() - timedelta(days=90)
-    RequestLog.objects.filter(timestamp__lt=ninety_days_ago).delete()
+    ninety_days_ago = datetime.utcnow() - timedelta(days=90)
+    queryset = RequestLog.objects.filter(timestamp__lt=ninety_days_ago)
+    DeleteQuery(RequestLog).delete_qs(queryset, using='default')
```

Comment:
Is this a documented method of django?
I can find this method in the latest version, which makes me worried it was deleted at some point

https://github.com/django/django/blob/master/django/db/models/sql/subqueries.py

Reply:
Did more digging, it was removed:

https://github.com/django/django/pull/11931/files#diff-0439081ad1f1b21a0112e02c47360a7711f340174d84f6c3ce155adf8b06c4caL44-L76

Reply:
Nice catch!

I used this because it is called by [`QuerySet._raw_delete()`](https://github.com/django/django/blob/master/django/db/models/query.py#L755), and I preferred not to use a private/protected method.

There have been calls to make `_raw_delete()` public ([discussion here](https://code.djangoproject.com/ticket/9519)) but it hasn't happened. How do you feel about using `_raw_delete()` here?

The alternative is raw SQL.